uzdevums-1/ExchangeRateApp/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement } from '@angular/core';
+import {MatButtonModule} from '@angular/material/button';
+import {MatTableModule} from '@angular/material/table';
+import {MatPaginatorModule} from '@angular/material/paginator';
+import {MatProgressBarModule} from '@angular/material/progress-bar';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { By } from '@angular/platform-browser';
+
+import { of } from 'rxjs';
+
+import { CurrencyTableComponent } from './currency-exchange-table/currency-table.component';
+import { DashboardComponent } from './dashboard.component';
+import { ExchangeService } from './shared/exchange.service';
+
+describe('DashboardComponent', () => {
+  let component: DashboardComponent;
+  let fixture: ComponentFixture<DashboardComponent>;
+  let exchangeService: ExchangeService;
+  let exchangeServiceStub: any;
+  let progressBar: DebugElement;
+  let getLatestRatesSpy: any;
+
+
+  beforeEach(async(() => {
```

Comment:
liels prieks, ka Tu neesi aizmirsis par testiem (it īpaši tāpēc, ka tie bija lūgti uzdevumā).